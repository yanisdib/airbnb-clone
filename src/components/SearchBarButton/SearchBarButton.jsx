import styled from 'styled-components';


export default styled.button`
    position: relative;
    height: 48px;
    width: 48px;
    max-width: 48px;
    appearance: none;
    background: transparent;
    background-color: #FF385C;
    color: #FFFFFF;
    display: inline-block;
    font-family: inherit;
    font-size: 16px;
    font-weight: 600;
    text-align: center;
    text-decoration: none;
    line-height: 16px;
    border: 0px;
    border-radius: 24px;
    margin: 0px;
    padding: 0px;
    cursor: pointer;
    outline: none;
    overflow: hidden;
    user-select: auto;

    svg {
        height: 23px;
    }
`;