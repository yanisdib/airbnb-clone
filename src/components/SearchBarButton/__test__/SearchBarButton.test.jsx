import { shallow } from 'enzyme';

import SearchBarButton from '../SearchBarButton';


describe('SearchBarButton component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<SearchBarButton>Search</SearchBarButton>))

    it('should render correctly without crashing', () => expect(wrapper).toMatchSnapshot())

    it('should have text as a children props', () => expect(wrapper.text()).toBe('Search'))
})