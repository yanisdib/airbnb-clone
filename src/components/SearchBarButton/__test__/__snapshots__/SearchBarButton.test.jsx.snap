// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBarButton component should render correctly without crashing 1`] = `
<button
  className="sc-bdvvtL eCRPxH"
>
  Search
</button>
`;
